packs/spells/Akashic_Assistant_4qXeaZLAtTxd5tNq.json

```diff
@@ -6,7 +6,7 @@
   "system": {
     "description": {
       "gm": "",
-      "value": "<p><strong>Connection</strong> akashic</p>\n<hr />\n<p>You summon a projection of one of the etheric entities that work as librarians and scribes within the Akashic Library to assist your target with a task. The projection appears floating next to the target and doesn't occupy any space. The assistant prepares to Aid the target as though it had its own reaction. The assistant uses a bonus equal to your modifier in Occultism for its rolls to Aid, regardless of what skill it is Aiding with. On each of your turns, the assistant prepares to Aid its target again. You can use a free action, which has the concentrate trait, to redirect the assistant to a different creature, changing the target of the spell. Otherwise, the assistant keeps aiding the same creature for the duration of the spell.</p>"
+      "value": "<p><strong>Connection</strong> akashic</p><hr /><p>You summon a projection of one of the etheric entities that work as librarians and scribes within the Akashic Library to assist your target with a task. The projection appears floating next to the target and doesn't occupy any space. The assistant prepares to [[/act aid statistic=occultism]]{Aid} the target as though it had its own reaction. The assistant uses a bonus equal to your modifier in Occultism for its rolls to Aid, regardless of what skill it is Aiding with. On each of your turns, the assistant prepares to Aid its target again. You can use a free action, which has the concentrate trait, to redirect the assistant to a different creature, changing the target of the spell. Otherwise, the assistant keeps aiding the same creature for the duration of the spell.</p>"
```

Comment:
```suggestion
      "value": "<p><strong>Connection</strong> akashic</p><hr /><p>You summon a projection of one of the etheric entities that work as librarians and scribes within the Akashic Library to assist your target with a task. The projection appears floating next to the target and doesn't occupy any space. The assistant prepares to [[/act aid statistic=occultism]] the target as though it had its own reaction. The assistant uses a bonus equal to your modifier in Occultism for its rolls to Aid, regardless of what skill it is Aiding with. On each of your turns, the assistant prepares to Aid its target again. You can use a free action, which has the concentrate trait, to redirect the assistant to a different creature, changing the target of the spell. Otherwise, the assistant keeps aiding the same creature for the duration of the spell.</p>"
```